src/main/AccessTokenBuilder.js

```diff
@@ -17,7 +17,7 @@ export default class AccessTokenBuilder {
 
         this._payload = {
             version: config.accessTokenVersion,
-            refId: Util.generateNonce(),
+            refId: Util.generateNonce().substr(0, 18),
```

Comment:
add optional length to generateNonce?

Reply:
I think this is clean enough, your suggestion is more like an optimization

Reply:
Will need to rewrite `generateNonce`, to generate given length of nonce